apps/charterafrica/src/components/Comments/Comments.js

```diff
@@ -0,0 +1,68 @@
+import { RichTypography } from "@commons-ui/core";
+import { Box, Grid, MenuItem, Select } from "@mui/material";
+import PropTypes from "prop-types";
+import React, { forwardRef } from "react";
+
+import Comment from "./Comment";
+
+import { neutral } from "@/charterafrica/colors";
+
+const Comments = forwardRef((props, ref) => {
+  const { comments } = props;
+  return (
+    <Box bgcolor="#FFF" ref={ref}>
+      <Grid
+        sx={{ p: 2.5 }}
+        alignItems="center"
+        justifyContent="space-between"
+        container
+      >
+        <Grid item>
+          <RichTypography variant="p3">
+            {comments.length} comments
+          </RichTypography>
+        </Grid>
+        <Grid
+          display="flex"
+          alignItems="center"
+          justifyContent="flex-end"
+          flex={1}
+          item
+        >
+          <RichTypography variant="p3">Sort by:</RichTypography>
```

Comment:
All these comments, sort by, most recent, etc., should be passed in via props for localization.